docs/source/table.rst

```diff
@@ -129,6 +129,17 @@
      - 252
      - :ikala:`\ `
 
+   * - Haydn op20
+     - - audio: ❌
```

Comment:
maybe `audio: N/A` makes more sense here, since the dataset is fully symbolic, rather than the audio being not available

Reply:
ok